dataverse-webapp/src/main/webapp/dataverse-editTextMessages.xhtml

```diff
@@ -60,6 +72,7 @@
                                 #{bundle['edittextmessages.startDateTime']} <span class="glyphicon glyphicon-asterisk text-danger" title="#{bundle.requiredField}"/>
                             </p:outputLabel>
                             <div class="col-sm-6 form-col-container">
+                                <p class="help-block">#{bundle['edittextmessages.startDateTime.format.tip']}</p>
```

Comment:
Apply the same for banners?

Reply:
My bad, missed banners in task description...